src/Agent.Worker/ExecutionContext.cs

```diff
@@ -621,6 +621,8 @@ private void JobServerQueueThrottling_EventReceived(object sender, ThrottlingEve
                     uriBuilder.Path += (Variables.System_TFCollectionUrl.EndsWith("/") ? "" : "/") + "_usersSettings/usage";
                     query["tab"] = "pipelines";
                     query["queryDate"] = queryDate;
+                    query["keywords"] = this.Variables.Build_Number;
```

Comment:
I think there might still be value in providing the user a link to the existing global pipelines resource utilization page, in addition to a link that scopes to build/release name and id.

Consider the case where the affected build/release is not actually the cause of the throttle, but is actually the victim (i.e. there's some other build/release that's actually causing the throttling). Opening the RU link scoped by name/id would result in no useful information for the throttling investigation.

My suggestion: provide two RU links -- the existing Pipelines link scoped only by query date, and another one scoped by query date, build/release name and id.

What do you think?